uninstall.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Fired when the plugin is uninstalled.
+ */
+ 
+/* If uninstall not called from WordPress, then exit. */
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	wp_die( sprintf( __( '%s should only be called when uninstalling the plugin.', 'wdp-plugin' ), '<code>' . __FILE__ . '</code>' ) );
+}
+
+
+/* Deleting FFPW widget option row in database */
+delete_option('widget_FFPW_Flexible_Featured_Post');
```

Comment:
This (delete_option) removes the row in the wp_options table of the database with the name of ‘widget_ffpw_flexible_featured_post’. This option was remaining in the database after deactivation and deletion of the plugin.